src/main/java/upm/softwaredesign/finalproject/order/Delivery.java

```diff
@@ -1,14 +1,14 @@
 package upm.softwaredesign.finalproject.order;
```

Comment:
This is Martin's team module? Have you consulted with him about this?

Reply:
The naming change was a must, I only updated the entity reference in the class. So there may be no conflicts or anything.

Reply:
Was it, because Martins module communicates only with Actor from Piotr's module.
Martin's module has no contact with yours :) So it should not know your classes.